xdsl/dialects/utils/format.py

```diff
@@ -35,6 +52,117 @@ def __init__(self, *operands: SSAValue | Operation):
         super().__init__(operands=[operands])
 
 
+def print_for_op_like(
+    printer: Printer,
+    lower_bound: SSAValue,
+    upper_bound: SSAValue,
+    step: SSAValue,
+    iter_args: Sequence[SSAValue],
+    body: Region,
+    bound_words: Sequence[str] = ["to"],
+    indvar_type: type[TypeAttribute] | None = None,
```

Comment:
It definitely wasn't clear to me what this did just from the name, perhaps `default_indvar_type` might be slightly better? (I see a description is in the docstring.

Reply:
yeah, I debated back and forth on the name; I'll amend.